test/core/_parser.scss

```diff
@@ -279,16 +279,16 @@
     $error-on-missing-keys: false !global;
   }
 
-  // @include it('Finds an explicitly terminated alias when strict mode is enabled') {
-  //   $error-on-missing-keys: true !global;
+  @include it('Finds an explicitly terminated alias when strict mode is enabled') {
+    $error-on-missing-keys: true !global;
 
-  //   @include assert-equal(
-  //     _a_replace($map, '#name\\ofthething'),
-  //     'N4M3!ofthething'
-  //   );
+    @include assert-equal(
+      _a_replace($map, '#name\\ofthething'),
```

Comment:
If I understand right, since `\` is already an escape character, users would have to use `\\`? I don't know if that's ironic, or a problem, or nice that it forces even more explicit intent…  😛 In any case, we'd want to make sure it's clear in the docs.

Reply:
It's actually because of how css & sass actually handle strings: [https://sass-lang.com/documentation/values/strings#escapes](https://sass-lang.com/documentation/values/strings#escapes).

Since the /character escapes any other character besides0-9 and A-F (the first point in that documentation item) writing `/#` is just parsed as`#`. You'd need to write it twice to escape itself, so to speak. So the parser reads `\\#` as `\#`. Valid point that it should be documented though.

Reply:
Yeah, that's what I was getting at. It's a Sass-escape character, so we have to Sass-escape it in order to generate our own escape character. A double-escape! Escaping all the way down!

Reply:
I don't know if there's some danger with that, but I kinda like that it means our escape is practically `\\` as far as users are concerned.

Reply:
I think it's fine, it means they can use unicode characters or escapes in their strings without any hassle. Maybe we should write some more tests at some point to cover any weird edge cases.